pkg/security/ebpf/c/exec.h

```diff
@@ -109,6 +109,15 @@ struct bpf_map_def SEC("maps/exec_count_bb") exec_count_bb = {
     .namespace = "",
 };
 
+struct bpf_map_def SEC("maps/task_in_coredump") tasks_in_coredump = {
+    .type = BPF_MAP_TYPE_LRU_HASH,
+    .key_size = sizeof(u64),
+    .value_size = sizeof(u64),
```

Comment:
Do we really need a value of type u64 since we only want a bool ? Maybe a u8 is enough ?

Reply:
applied